### PR TITLE
docs: updated calendar documentation

### PR DESCRIPTION
**Pull request by jessmetivier**
## Description
Updating documentation for **calendar component**.

### CODE REVIEW

File: stories/calendar/calendar.stories.js [hunk 1]
```diff
@@ -4,75 +4,96 @@ export default {
...
-    - <code class="docs-code">.fd-calendar\\_\\_close-button</code> button inside dialog header for calendar on mobile devices
-  - <code class="docs-code">.fd-calendar\\_\\_content</code> calendar body container
-    - <code class="docs-code">.fd-calendar\\_\\_table</code> groups table elements representing calendar items
-      - <code class="docs-code">.fd-calendar\\_\\_group</code> groups calendar rows with the same destination
-        - <code class="docs-code">.fd-calendar\\_\\_row</code> groups calendar grid items in one row
-          - <code class="docs-code">.fd-calendar\\_\\_item</code> represents calendar grid item
-            - <code class="docs-code">.fd-calendar\\_\\_text</code> represents content of calendar grid item
-          - <code class="docs-code">.fd-calendar\\_\\_special-day</code> represents calendar grid item with special marker
+- You want the user to select a single date, multiple days, whole week(s), or a date range.
+- You want to display multiple months at once.
```
Comment (mikerodonnell89) [inline stories/calendar/calendar.stories.js:17]: or years as well

File: stories/calendar/calendar.stories.js [hunk 2]
```diff
@@ -288,7 +309,8 @@ export const days = () => `
 
 days.parameters = {
     docs: {
-        iframeHeight: 400
+        iframeHeight: 400,
+        storyDescription: 'The calendar component can display days of the month by adding the <code>–days</code> modifier class to the element.'
```
Comment (mikerodonnell89) [inline stories/calendar/calendar.stories.js:313]: I think here you mean "--dates" on the content element, "fd-calendar__content--dates" correct?

File: stories/calendar/calendar.stories.js [hunk 3]
```diff
@@ -371,7 +393,8 @@ export const months = () => `
 
 months.parameters = {
     docs: {
-        iframeHeight: 400
+        iframeHeight: 400,
+        storyDescription: 'The calendar component can display months in a year by adding the <code>–months</code> modifier class to the element.'
```
Comment (mikerodonnell89) [inline stories/calendar/calendar.stories.js:397]: Same thing here ("fd-calendar__content--months")

File: stories/calendar/calendar.stories.js [hunk 4]
```diff
@@ -605,11 +628,12 @@ export const years = () => `
 
 years.parameters = {
     docs: {
-        iframeHeight: 700
+        iframeHeight: 700,
+        storyDescription: 'The calendar component can display a range of years in various ways: short-term, long-term, year ranges etc. Add the <code>–years</code> modifier class to the element, and adjust the number of columns, rows, and content to fit your use case.'
```
Comment (mikerodonnell89) [inline stories/calendar/calendar.stories.js:632]: and here

File: stories/calendar/calendar.stories.js (hunk 2 above)
Comment (jessmetivier) [reply stories/calendar/calendar.stories.js:313]: @mikerodonnell89  That's correct, if I added **"to the <code>fd calendar__content</code> element"** is that more understandable? Or can I simply add **"content element"** at the end of the sentence?

File: stories/calendar/calendar.stories.js (hunk 1 above)
Comment (jessmetivier) [reply stories/calendar/calendar.stories.js]: Good catch! Added **years** to the sentence.

File: stories/calendar/calendar.stories.js (hunk 2 above)
Comment (jessmetivier) [reply stories/calendar/calendar.stories.js]: I just updated the sentences, let me know if they make sense now :)

File: stories/calendar/calendar.stories.js (hunk 3 above)
Comment (jessmetivier) [reply stories/calendar/calendar.stories.js]: Done. Please review

File: stories/calendar/calendar.stories.js (hunk 4 above)
Comment (jessmetivier) [reply stories/calendar/calendar.stories.js]: Done. Please review

**Review by JKMarkowski — APPROVED** [review]
LGTM

**Review by salarenko — APPROVED** [review]

**Pull request merged**